src/Tasks/GetReferenceAssemblyPaths.cs

```diff
@@ -248,7 +248,7 @@ private IList<String> GetPaths(string rootPath, string targetFrameworkFallbackSe
             {
                 // No reference assembly paths could be found, log an error so an invalid build will not be produced.
                 // 1/26/16: Note this was changed from a warning to an error (see GitHub #173).
-                if (pathsToReturn.Count == 0)
+                if (pathsToReturn.Count == 0 && NativeMethodsShared.IsWindows)
```

Comment:
Is the right condition here `IsWindows`, or `#if NET`? Does the find-targeting-packs code work on Windows/net9?